.github/workflows/build.yml

```
@@ -139,15 +122,133 @@ jobs:
+                  chmod +x Linux-X64/bin/zrc Linux-X64/bin/zircop
+                  chmod +x Linux-ARM64/bin/zrc Linux-ARM64/bin/zircop
+                  chmod +x macOS-X64/bin/zrc macOS-X64/bin/zircop
+                  chmod +x macOS-ARM64/bin/zrc macOS-ARM64/bin/zircop
+            - name: Create tar archives
+              run: |
+                  tar -czf zrc-linux-x64.tar.gz -C Linux-X64 .
+                  tar -czf zrc-linux-arm64.tar.gz -C Linux-ARM64 .
+                  tar -czf zrc-macos-x64.tar.gz -C macOS-X64 .
+                  tar -czf zrc-macos-arm64.tar.gz -C macOS-ARM64 .
+            - name: Update `nightly` release
+              uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)